TESTS/assumptions/SPI/SPI.cpp

```diff
@@ -1,3 +1,8 @@
+// check if SPI is supported on this device
+#if !DEVICE_SPI
+    #error SPI is not supported on this platform, add 'DEVICE_SPI' definition to your platform.
```

Comment:
I double checked compilation on K64F and DELTA_DFCM_NNN50 and it looked good to me.

Perhaps we should add some Travis CI checks to double check compilation.... :stuck_out_tongue_winking_eye: